ts/sagas/startup/loadSessionInformationSaga.ts

```diff
@@ -21,15 +23,20 @@ import { SagaCallReturnType } from "../../types/utils";
  *        a saga.
  */
 export function* loadSessionInformationSaga(
-  getSession: TypeofApiCall<GetSessionStateT>
+  getSession: ReturnType<typeof BackendClient>["getSession"]
 ): IterableIterator<Effect | Option<PublicSession>> {
   // Call the Backend service
   const response: SagaCallReturnType<typeof getSession> = yield call(
```

Comment:
@Undermaken note that `response` is never `undefined`, the previous purpose of `undefined` (i.e. unable to decode) is now mapped to `Left<Errors>` (decoding errors)